plugin/Installer/DatabaseChanges.php

```diff
@@ -274,6 +281,7 @@ private function getCreateQueryForms()
 			`radius` INT( 10 ) NULL DEFAULT NULL,
 			`geo_order` VARCHAR( 255 ) NOT NULL DEFAULT 'street,zip,city,country,radius',
 			`show_estate_context` tinyint(1) NOT NULL DEFAULT '0',
+			`page_shortcode` tinytext NOT NULL,
```

Comment:
isn't this duplicate information? This is the name with some constant string left and right. Totally reproducible.